idd/Energy+.idd.in

```diff
@@ -38873,34 +38870,22 @@ AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan,
        \minimum 0.0
        \maximum 1.0
        \note fraction of cooling air flow rate
-  A3,  \field Air Inlet Node Name
```

Comment:
Hmm, this is surprising.  This leaves the terminal unit with *no* node names at all.  That makes it different from other terminal units. I realize this has been marked deprecated for a long time, but wondering if we want to keep it that way.  Will check the history.